README.md

```diff
@@ -5,6 +5,7 @@
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Key Features](#key-features)
+
 - [👤 Author](#authors)
 - [📝 License](#license)
```

Comment:
- [ ]  Good job adding the recommended readme as suggested by the previous reviewer 👍🏽 . Please do not delete any section of the microverse template (FAQ section is optional) as I see you did that, but rather, kindly edit all the sections of your readme and add information relating to your project so that your repo will look more professional 👍🏽.